compaction/compaction_manager.hh

```diff
@@ -301,6 +301,9 @@ private:
     utils::updateable_value<float> _static_shares;
     serialized_action _update_compaction_static_shares_action;
     utils::observer<float> _compaction_static_shares_observer;
+    uint64_t validation_errors_segregate = 0;
+    uint64_t validation_errors_validate = 0;
```

Comment:
I don't think it is interesting to separate the two. These are very rare, manually started operations.

Reply:
I think just a general `validation_error`, that you increase for both `scrub` and `validatate` compaction is sufficient. These two operations will never run in parallel, so the operator will always be able to tell where the validation errors are coming from.